src/Artesian/MarketData/MarketDataService.py

```diff
@@ -113,6 +114,91 @@ def readCurveRange(
             )
         )
 
+    async def searchFacetAsync(
+        self: MarketDataService,
+        page: int,
+        pageSize: int,
+        searchText: str = None,
+        filters: Optional[Dict[str, List[str]]] = None,
```

Comment:
this is correct, the typing, but doesn't match the example.

the example is a list of records, each with Key an Value, not a Dictionary of string,string[]

The typing is correct, is the example that is wrong.